### ISSUE TITLE
Sort out how to delete an account with a completely off-chain and offline signature

### ISSUE THREAD

**Issue by natanasow** [hashgraph/hedera-json-rpc-relay#2648]
### Problem

Currently, we have an account that we want to delete but the AccountDeleteTransaction signing must be completely offchain. The following execution must be with another account and in the `validStart` timeframe.

### Solution

There are 2 actions we need to cover:
- Add a script for signing the transaction off-chain and offline. The script's output should be signed transaction bytes that we should use in the "executor" job.
- Add an executor script that should track the transaction's `validStart` time and must pass the transaction to the nodes when the `validStart` occurs.

**Comment by Nana-EC** [comment]
This issue will be our 2nd attempt to provide a work around for https://github.com/safe-global/safe-singleton-factory/issues/206 
With this offline scenario Safe can sign the HAPI transaction to support the account nonce reset and then allow anyone to submit it to Hedera for them.
This should unblock the deployment of the SAFE factory contract.

**Comment by Nana-EC** [comment]
Closing in favour of https://github.com/hashgraph/hedera-json-rpc-relay/issues/3298
